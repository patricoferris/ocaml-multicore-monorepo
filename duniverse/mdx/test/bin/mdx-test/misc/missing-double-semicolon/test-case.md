#files that contain top level blocks missing ;; should trigger a single warning

```ocaml
# let x = 42
```

```ocaml
# let y = 23
```
