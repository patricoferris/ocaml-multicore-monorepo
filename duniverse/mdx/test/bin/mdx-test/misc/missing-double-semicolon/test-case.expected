Warning: OCaml toplevel block without trailing ;; detected in file 'test-case.md'.
Non-semicolon terminated phrases are deprecated.
In MDX 3.0 support for toplevel blocks without ;; will be removed.
